packages/kokkos-kernels/src/sparse/KokkosSparse_spadd.hpp

```diff
@@ -69,15 +69,17 @@ namespace Experimental {
   struct SortedCountEntries
   {
     SortedCountEntries(
-        const typename ARowPtrsT::const_type Arowptrs_,
-        const AColIndsT Acolinds_,
-        const typename BRowPtrsT::const_type Browptrs_,
-        const BColIndsT Bcolinds_,
-        CRowPtrsT Crowcounts_) :
+        const typename ARowPtrsT::const_type& Arowptrs_,
+        const AColIndsT& Acolinds_,
+        const typename BRowPtrsT::const_type& Browptrs_,
+        const BColIndsT& Bcolinds_,
+        CRowPtrsT& Crowcounts_) :
```

Comment:
Why is this one a nonconst reference?  If these are all `Kokkos::View` objects and if you're not reallocating them, then you can pass them either by value or by const reference.